src/javascript/app/pages/user/account/settings/personal_details.js

```diff
@@ -199,6 +281,29 @@ const PersonalDetails = (() => {
                 { selector: '#chk_tax_id',     validations: is_financial ? [['req', { hide_asterisk: true, message: localize('Please confirm that all the information above is true and complete.') }]] : '', exclude_request: 1 },
             ];
 
+            // Push validations for changeable fields.
+            if (has_changeable_fields) {
```

Comment:
I don't think we need an exception for pushing these to validations, we can include them in the validations list, since validator will handle if the selector for these doesn't exist or is invisible

Reply:
Removing if statement. But I think letting the `forEach` loop stay, since it is more dynamic (with a few exceptions).